targets/ChibiOS/_WiFi/inventek/es_wifi_io_chibios.c

```
@@ -0,0 +1,393 @@
+        (rxBuffer[3] != '\n') || (rxBuffer[4] != '>') || (rxBuffer[5] != ' '))
+    {
+        return ES_WIFI_STATUS_ERROR;
+    }
+
+    return ES_WIFI_STATUS_OK;
+}
+
+// DeInitialize the SPI
+// @param  None
+// @retval None
+int8_t SPI_WIFI_DeInit(void)
```

Cppcheck (2.21.0)

[style] The function 'SPI_WIFI_DeInit' is never used.

(unusedFunction)